chef/cookbooks/crowbar/templates/default/crowbar-evacuate-host.sh.erb

```diff
@@ -41,17 +41,20 @@ blockmigrate=""
 <% end %>
 if ! nova host-evacuate-live "$host" $blockmigrate; then
     log "Live migration of instances from host $host has failed!"
-    touch $UPGRADEDIR/crowbar-evacuate-host-failed
+    echo 2 > $UPGRADEDIR/crowbar-evacuate-host-failed
     exit 2
 fi
 
-# FIXME: here we need checks for potential errors during the evacuation
-
 while nova list --all-tenants --host "$host" --fields host,status 2>/dev/null | grep -e ACTIVE -e MIGRATING | grep -q "$host"; do
     log "There is still some VM running at $host..."
     sleep 10
 done
 
+if nova list --all-tenants --host "$host" --fields status | grep -q ERROR; then
```

Comment:
It's `Task State` that is `migrating`. So I think it's gonna stay as `migrating` even when Status changes to `ERROR`.